ojo-provider-config/currency-pairs.toml

```diff
@@ -164,13 +158,6 @@ providers = [
 ]
 quote = "OSMO"
 
-[[currency_pairs]]
```

Comment:
We might want to put these configs in a separate PR just to avoid conflicts when we merge this into the umee branch

But, those conflicts would be pretty easy so up to you 🤷

Reply:
Not my first conflicts on an umme backport :). No problem, easier to have one review and not leave the repo in a broken state.